modules/apps/commerce/commerce-shipping-engine-remote/src/main/java/com/liferay/commerce/shipping/engine/remote/internal/portlet/action/EditRemoteCommerceShippingMethodConfigurationMVCActionCommand.java

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.commerce.shipping.engine.remote.internal.portlet.action;
+
+import com.liferay.commerce.constants.CommercePortletKeys;
+import com.liferay.commerce.shipping.engine.remote.internal.constants.RemoteCommerceShippingEngineConstants;
+import com.liferay.portal.kernel.exception.PortalException;
+import com.liferay.portal.kernel.portlet.bridges.mvc.BaseMVCActionCommand;
+import com.liferay.portal.kernel.portlet.bridges.mvc.MVCActionCommand;
+import com.liferay.portal.kernel.settings.GroupServiceSettingsLocator;
+import com.liferay.portal.kernel.settings.ModifiableSettings;
+import com.liferay.portal.kernel.settings.Settings;
+import com.liferay.portal.kernel.settings.SettingsFactory;
+import com.liferay.portal.kernel.util.ParamUtil;
+import com.liferay.portal.kernel.util.Portal;
+import com.liferay.portal.kernel.util.Validator;
+
+import javax.portlet.ActionRequest;
+import javax.portlet.ActionResponse;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Ivica Cardic
+ */
+@Component(
+	enabled = false, immediate = true,
+	property = {
+		"javax.portlet.name=" + CommercePortletKeys.COMMERCE_SHIPPING_METHODS,
+		"mvc.command.name=editRemoteCommerceShippingMethodConfiguration"
```

Comment:
New mvc.command.name format https://github.com/liferay/liferay-portal/blob/master/modules/util/source-formatter/src/main/resources/documentation/checks/mvc_command_name_check.markdown

/{javax.portlet.name}/{mvcActionCommand}
```suggestion
		"mvc.command.name=/commerce_shipping/edit_remote_commerce_shipping_method_configuration"
```